packages/cloudflare/README.md

```diff
@@ -222,21 +240,108 @@ import cloudflare from '@astrojs/cloudflare';
 export default defineConfig({
   output: 'server',
   adapter: cloudflare({
-+   runtime: { mode: 'local' },
++   runtime: {
++     mode: 'local',
++     type: 'pages',
++     bindings: {
+        // example of a KV binding
++       "URL": {
++         type: "var",
++         id: "https://example.com",
++       },
+        // example of a KV binding
++       "KV": {
++         type: "kv",
++         id: "KV",
++       },
+        // example of a D1 binding
++       "D1": {
++         type: "d1",
++         database_name: "D1"
++         preview_database_id: "D1",
++         database_id: "D1",
++       },
+        // example of a R2 binding
++       "R2": {
++         type: "r2",
++         bucket_name: "R2",
++       },
+        // example of a Durable Object binding
++       "DO": {
++         type: "durable-object",
++         className: "DO",
++       },
++     },
++   },
```

Comment:
**[DOCS]**
We got feedback that we should document how to configure bindings, therefore here I've gone with one example per supported binding. WDYT?

Reply:
[DOCS]

These look very helpful! My only concern is not showing valid code. If you are only ever going to configure ONE of these at a time, then this example is misleading. (Admittedly, I don't know this, so I'm just checking)

It's not the biggest deal, but I would caution about showing multiple configs in the same file if this isn't how it works without some kind of warning that you'll only actually use one of these at the same time. If it is possible to configure multiple, then great!

Reply:
You can configure any of them in any combination and any amount. 
All of the following examples are valid. My only concern now is that it is not clear that you can configure it like this. What is you impression? @sarah11918 

- `var + var + var`
- `var + var + D1 + KV + KV`
- `var + D1 + var + KV + R2 + var`